src/Native/Unix/Common/pal_config.h.in

```diff
@@ -32,7 +32,7 @@
 #cmakedefine01 HAVE_TIOCGWINSZ
 #cmakedefine01 HAVE_SCHED_GETAFFINITY
 #cmakedefine01 HAVE_SCHED_SETAFFINITY
-#cmakedefine01 HAVE_ARC4RANDOM
+#cmakedefine01 HAVE_ARC4RANDOM_BUF
```

Comment:
Please undo this change or do it upstream first (to avoid future merge conflicts)

Reply:
These are now in corefx:
https://github.com/dotnet/corefx/commit/8a05b3f5f3947341e1867210a791a5c8a617cde6